### PR TITLE
[WIP]Update script http 009

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

**Comment by istoimenova** [comment]
[WIP] - Clarified with Igor Kovalenko

File: test_scripts/Polices/build_options/009_ATF_P_TC_Policy_Table_Snapshot_Creation_HTTP.lua [hunk 1]
```diff
@@ -51,29 +51,26 @@ end
...
-      is_test_passed = testCasesForPolicyTableSnapshot:verify_PTS(true,
-              { config.application1.registerAppInterfaceParams.appID, config.application2.registerAppInterfaceParams.appID, },
-              {config.deviceMAC},
-              {""},
-              "print")
-    end
-  end)
-  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "HTTP"})
+      if(data.payload.requestType == "HTTP") then
+        if(data.binaryData == nil) then
```
Comment (dboltovskyi) [inline test_scripts/Polices/build_options/009_ATF_P_TC_Policy_Table_Snapshot_Creation_HTTP.lua]: @istoimenova, Please add additional check on blank value ("")

Comment (istoimenova) [inline test_scripts/Polices/build_options/009_ATF_P_TC_Policy_Table_Snapshot_Creation_HTTP.lua]: @dboltovskyi, done in 373034b4c6bad6115376918d190416e17feb6b13

**Pull request merged**